cupy/linalg/norms.py

```diff
@@ -69,36 +69,24 @@ def norm(x, ord=None, axis=None, keepdims=False):
         elif ord == 0:
             # Zero norm
             # Convert to Python float in accordance with NumPy
-            return (x != 0).sum(axis=axis, keepdims=keepdims, dtype='d')
+            return (x != 0).astype(x.real.dtype).sum(axis=axis, keepdims=keepdims)
```

Comment:
Cloud you fix this line.
`./cupy/linalg/norms.py:72:80: E501 line too long (82 > 79 characters)`

Reply:
Fixed.